Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2743,10 +2743,43 @@ bool CGContextIsPointInPath(CGContextRef context, bool eoFill, CGFloat x, CGFloa
 struct __CGBitmapContext : CoreFoundation::CppBase<__CGBitmapContext, __CGContext> {
     woc::unique_cf<CGImageRef> _image;
 
+    void* _data;
```

Comment:
nit: I know this one has been around a bit longer and CG has been using it, but I like the m_* naming convention for C++ structs/classes since it makes it more obvious what's C++ and what's objc.